helm/korifi/controllers/cf_roles/cf_root_namespace_user.yaml

```diff
@@ -46,6 +46,7 @@ rules:
   - cfservicebrokers
   verbs:
   - list
+  - delete
```

Comment:
Adding the `delete` permission here would allow all authenticated users to delete offerings. According to the CF API spec, only admins should be allowed to do that. Therefore, this permission should be added to the [`cf_admin` role](https://github.com/cloudfoundry/korifi/blob/main/helm/korifi/controllers/cf_roles/cf_admin.yaml)